changelog.d/20240604_170320_chris_document_443_default_switch.rst

```diff
@@ -0,0 +1,7 @@
+Changed
+^^^^^^^
+
+- The Compute hosted services now tell the SDK to connect to AMQP over port 443 by
+  default. This means that ``Executor(endpoint_id=..., amqp_port=443)`` is now redundant.
+  To go back to the old port of 5671, set ``Executor.amqp_port`` to 5671 via the
```

Comment:
I think "To go back to the old" may be confusing without context, as not everyone is upgrading, so 'back to' may not always make senes.

How about "To use the standard AMQP port, set ...."

Reply:
Rewrote the changelog to be more friendly to end users